.gitlab-ci.yml

```diff
@@ -839,12 +839,15 @@ workflow:
 .on_security_agent_changes_or_manual:
   - <<: *if_main_branch
   - changes:
-      - pkg/ebpf/**/*
-      - pkg/security/**/*
-      - test/kitchen/site-cookbooks/dd-security-agent-check/**/*
-      - test/kitchen/test/integration/security-agent-test/**/*
-      - test/kitchen/test/integration/security-agent-stress/**/*
-      - .gitlab/functional_test/security_agent.yml
+      paths:
+        - pkg/ebpf/**/*
+        - pkg/security/**/*
+        - pkg/eventmonitor/**/*
+        - test/kitchen/site-cookbooks/dd-security-agent-check/**/*
+        - test/kitchen/test/integration/security-agent-test/**/*
+        - test/kitchen/test/integration/security-agent-stress/**/*
+        - .gitlab/functional_test/security_agent.yml
+      compare_to: main
```

Comment:
I think that, in the long-term, we'd like this to use a variable (that way, the branch would only need to be set once for every job that uses this). That is not possible yet according to this issue: https://gitlab.com/gitlab-org/gitlab/-/issues/369916, so could we add a TODO here?

Reply:
I opened https://gitlab.com/gitlab-org/gitlab/-/merge_requests/126860 to implement variable expansion upstream